contrib/init/systemd/docker.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=Docker Application Container Engine
 Documentation=https://docs.docker.com
-After=network.target docker.socket firewalld.service
+After=network-online.target docker.socket firewalld.service
```

Comment:
Can you update the PR with those changes? While you're doing that, can you squash your commits, so that there's a single commit in this PR? (some information on how to do that can be found in the [contributors guide](https://docs.docker.com/opensource/workflow/work-issue/#/pull-and-rebase-frequently))